_data/simple-icons.json

```
@@ -8527,6 +8527,16 @@
             "hex": "E1171E",
             "source": "https://picnic.app/nl/feestdagen/"
         },
+        {
+            "title": "Pico.CSS",
+            "hex": "12191F",
+            "source": "https://user-images.githubusercontent.com/90356347/215237356-83b5d992-65be-464e-bccd-63ff62ec5579.svg",
+            "aliases": {
+                "aka": [
+                    "Picodotcss"
+                ]
+            }
```

```suggestion
            "source": "https://picocss.com"
```

There's an SVG in the header of the site, and we don't need to worry about the alias.